Constrictor.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
   s.swift_version = "4.2"
   s.name         = "Constrictor"
-  s.version      = "4.0.0"
+  s.version      = "4.0.1"
```

Comment:
this should be major

Reply:
100% agree but 4.0.0 was just released yesterday and nobody is currently using. With this in mind I don't want to bump another major due to a minor fix.

Reply:
I do not agree with you, but it's just an opinion